index.js

```diff
@@ -128,40 +149,35 @@ exports.empty_return_path = function (next, connection) {
   // validate that the Return-Path header is empty, RFC 3834
 
   const rp = transaction.header.get('Return-Path')
-  if (!rp) {
+  if (!rp || rp === '<>') {
     transaction.results.add(this, { pass: 'empty_return_path' })
     return next()
   }
 
-  if (rp === '<>') {
-    transaction.results.add(this, { pass: 'empty_return_path' })
-    return next()
+  transaction.results.add(this, { fail: 'empty_return_path', emit: true })
+
+  if (this.cfg.reject.empty_return_path) {
+    return next(DENY, 'bounce with non-empty Return-Path (RFC 3834)')
   }
 
-  transaction.results.add(this, { fail: 'empty_return_path', emit: true })
-  return next(DENY, 'bounce with non-empty Return-Path (RFC 3834)')
+  return next()
```

Comment:
```suggestion
  next()
```

Again, avoid useless use of `return`.